tests/yam/agama/agama.pm

```diff
@@ -25,6 +25,7 @@ sub run {
     my $test = get_required_var('AGAMA_TEST');
     my $reboot_page = $testapi::distri->get_reboot_page();
 
+    sleep(300);
```

Comment:
please, do not add sleeps with PR without WIP, they are not allowed in this repo, it is written somewhere about how to contribute (I don't remember where though).